nilearn/glm/first_level/first_level.py

```diff
@@ -220,6 +220,38 @@ def run_glm(Y, X, noise_model='ar1', bins=100,
         results = {0.0: ols_result}
 
     return labels, results
+  
+  
+def _check_trial_type(events):
+  
+    """ Raises warning if the events.tsv file
```

Comment:
```suggestion
    """ Raises a warning if the events.tsv file
```

Reply:
Actually to make sure that flake8 will not complain, we should use the imperative mood.

```
"""Raise a warning if the events.tsv file
```